### ISSUE TITLE
Is it possible to make a release for Raspberry Pi

### ISSUE THREAD

**Issue by emmtte** [claudiodangelis/qrcp#124]
Hi,

Thanks for your easy tool.
But, is it possible to make a release for a Raspberry Pi?

**Comment by claudiodangelis** [comment]
Hello @emmtte, thanks for pointing this out. It should be very easy to add that, it's just a matter of adding a couple lines to our GoReleaser config file. 

I'm posting this link as a note to self: https://goreleaser.com/build/

Will have a look at it later today, thanks!

**Comment by snowdream** [comment]
Please support Raspberry Pi 3 and newer Pi.

**Comment by claudiodangelis** [comment]
Hello @snowdream and @emmtte, I created a test release on this test repository: 
https://github.com/claudiodangelis/qrcp-test-arm/releases/tag/0.0.0

I'm not too much of an expert on this field, but the correct mapping should be

| Raspberry Model |  Binary |
| --- | --- |
| Rpi 1 | qrcp_0.6.1_linux_armv6.tar.gz |
| Rpi 2 | qrcp_0.6.1_linux_armv7.tar.gz |
| Rpi 3, 4 | qrcp_0.6.1_linux_arm64.tar.gz |

Thank you very much.

**Comment by claudiodangelis** [comment]
If you could test it whenever you have time it would really helpful. I do have a few Rpi boards but will only be possible for me to test it in a few days.

Thanks.

**Comment by snowdream** [comment]
> If you could test it whenever you have time it would be really helpful. I do have a few Rpi boards but will only be possible for me to test it in a few days.
> 
> Thanks.

For Rpi 3 b+, with CentOS 8, it use armv7l version.

```bash
uname -a
Linux localhost 4.19.72-300.el8.armv7hl #1 SMP Fri Sep 20 17:55:47 UTC 2019 armv7l armv7l armv7l GNU/Linux
```

```bash
 cat /etc/os-release
NAME="CentOS Linux"
VERSION="8 (AltArch)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="CentOS Linux 8 (AltArch)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:8"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-8"
CENTOS_MANTISBT_PROJECT_VERSION="8"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="8"
```

When i use qrcp_0.6.1_linux_armv7.tar.gz, It works like a charm.

```bash
 qrcp LICENSE 
only one interface found: wlan0, using this one
Scan the following URL with a QR reader to start the file transfer:
http://192.168.2.144:34639/send/x07r
██████████████████████████████████
█████████████████████████████████████
████ ▄▄▄▄▄ ██ ▀ █   ▀█▄▀██ ▄▄▄▄▄ ████
████ █   █ █ ▄▄▀▄▄▀ █▀██▄█ █   █ ████
████ █▄▄▄█ █ ▄▄  ██▄▄▄ ▀██ █▄▄▄█ ████
████▄▄▄▄▄▄▄█ █▄█ █ █ █ ▀ █▄▄▄▄▄▄▄████
████▄█  ▄▄▄▀▀██▄ ▀▀▀██▄ █▀   ▄▄█▀████
████▄▄██ █▄ █▀▀▀▄ ▄ ▀  ▀▀▄▀ ███▀█████
████▄▄ █  ▄ ▀▀ ▄██▄▄▀▄▀   ▀█▄▄ █▀████
████▄▄██▀▄▄▀█▀ ▀▀█▀█▀ ▀▀  ▀█ █▄▀█████
████  ▄██▀▄▄▄▀▀█ ▀▀█▀▀▀▄██▄█▀▀ █▀████
████ ██▀ ▄▄▀▄▄█▄█ ▄ █   ▄█▄█ ▄▄▀█████
████▄█▄▄█▄▄█▀ ▀ ▀█▄▄▀▀█▄ ▄▄▄ ▀   ████
████ ▄▄▄▄▄ █▀▀█▀▄█▀ █▀▀█ █▄█ ▄▄█▀████
████ █   █ █ ▀█ ▄█▀▄ ▀▀ ▄  ▄ ▄█▀█████
████ █▄▄▄█ █▄█▄█ ▀ █▀█ ▀▀▀█▄▀██ █████
████▄▄▄▄▄▄▄█▄██▄▄███▄███▄█▄▄█████████
█████████████████████████████████████
▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
```

**Comment by emmtte** [comment]
Working well for me too
And for Raspberry Pi 2 & 3 we need to use  qrcp_0.6.1_linux_armv7.tar.gz if 32bits OS
You can close the ticket.
Well done and fast resolve

**Comment by claudiodangelis** [comment]
This will be available in release 0.6.2 which is in the works, thanks for reporting!
